search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -11,11 +11,30 @@
 // WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 // License for the specific language governing permissions and limitations under
 // the License.
-
-// The colors assigned depending on if the sentiment score is positive or 
+ 
```

Comment:
Remove empty space.

Reply:
I thought we usually add one empty line between the copyright license and the code?